### PR TITLE
First release to new name

### PR DESCRIPTION
**Pull request by robin-checkmk**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

The collection resided under the namespace `tribe29`and name `checkmk`.

## What is the new behavior?

Following the rebranding of the company from `tribe29 GmbH` to `Checkmk GmbH`, we need to reflect this with our collection. Hence, this pull request finishes the renaming of the collection.

The collection now resides under the namespace `checkmk` and name `general`.

## Other information

https://github.com/ansible-community/ansible-build-data#renaming-a-collection
https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html

### CODE REVIEW

**Comment by robin-checkmk** [comment]
Molecule tests seem broken again due to upstream issues outside our control. Will ignore those for now.

**Comment by robin-checkmk** [comment]
I need to finish the renaming, so I will ignore the tests for now and revisit the findings later.

File: plugins/modules/rule.py [hunk 1]
```diff
@@ -445,11 +445,11 @@ def run_module():
...
                     deprecated_aliases=[
-                        dict(
-                            name="folder",
-                            collection_name="tribe29.checkmk",
-                            version="1.0.0",
-                        ),
+                        {
+                            "name": "folder",
+                            "collection_name": "checkmk.general",
+                            "version": "3.0.0",
```
Comment (lgetwan) [inline plugins/modules/rule.py:451]: Why 3.0.0? I thought we'd start with 1.0.0 after renaming.

File: SUPPORT.md [hunk 2]
```diff
@@ -28,4 +28,5 @@ Collection Version | Checkmk Versions | Ansible Versions | Remarks
 0.21.0 | 2.1.0p27, 2.0.0p35, 2.2.0b6 | 2.12, 2.13, 2.14 | None
 0.22.0 | 2.1.0p27, 2.0.0p35, 2.2.0b6 | 2.12, 2.13, 2.14 | None
 0.23.0 | 2.1.0p28, 2.0.0p36, 2.2.0b8 | 2.13, 2.14, 2.15 | None
-1.0.0 | 2.1.0p28, 2.0.0p36, 2.2.0b8 | 2.13, 2.14, 2.15 | None
\ No newline at end of file
+1.0.0 | 2.1.0p28, 2.0.0p36, 2.2.0b8 | 2.13, 2.14, 2.15 | None
+2.0.0 | 2.0.0p36, 2.1.0p28, 2.2.0p1 | 2.13, 2.14, 2.15 | None
```
Comment (lgetwan) [inline SUPPORT.md:32]: Why 2.0.0? I thought we'd start with 1.0.0 after renaming.

File: galaxy.yml [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
+namespace: checkmk
 
 # The name of the collection. Has the same character restrictions as 'namespace'
-name: checkmk
+name: general
 
 # The version of the collection. Must be compatible with semantic versioning
 
-version: 1.0.0
+version: 2.0.0
```
Comment (lgetwan) [inline galaxy.yml:13]: Why 2.0.0? I thought we'd start with 1.0.0 after renaming.

File: plugins/modules/host.py [hunk 4]
```diff
@@ -307,8 +307,8 @@ def run_module():
             deprecated_aliases=[
                 {
                     "name": "host_name",
-                    "date": "2024-01-01",
-                    "collection_name": "tribe29.checkmk",
+                    "collection_name": "checkmk.general",
+                    "version": "3.0.0",
```
Comment (lgetwan) [inline plugins/modules/host.py:311]: Why 3.0.0? I thought we'd start with 1.0.0 after renaming.

File: plugins/modules/host_group.py [hunk 5]
```diff
@@ -343,8 +343,8 @@ def run_module():
             deprecated_aliases=[
                 {
                     "name": "host_group_name",
-                    "date": "2023-03-01",
-                    "collection_name": "tribe29.checkmk",
+                    "collection_name": "checkmk.general",
+                    "version": "3.0.0",
```
Comment (lgetwan) [inline plugins/modules/host_group.py:347]: Why 3.0.0? I thought we'd start with 1.0.0 after renaming.

File: plugins/modules/host_group.py [hunk 6]
```diff
@@ -357,8 +357,8 @@ def run_module():
             deprecated_aliases=[
                 {
                     "name": "host_groups",
-                    "date": "2023-03-01",
-                    "collection_name": "tribe29.checkmk",
+                    "collection_name": "checkmk.general",
+                    "version": "3.0.0",
```
Comment (lgetwan) [inline plugins/modules/host_group.py:361]: Same...

**Review by lgetwan — COMMENTED** [review]
There are some versioning strings that I don't get correctly.

**Pull request merged**